package.json

```diff
@@ -12,13 +12,13 @@
     "build": "tsc --build tsconfig.app.json --force",
     "build:tests": "tsc --project tsconfig.app.json --outDir dist-test --declarationDir dist && tsc --build tsconfig.test.json",
     "clean": "rm -rf dist* *.tsbuildinfo",
-    "test": "npm run build:tests && node --test --experimental-test-snapshots",
-    "test-update-snapshots": "npm run build:tests && node --test --test-update-snapshots --experimental-test-snapshots"
+    "test": "npm run build:tests && node --test --no-experimental-strip-types",
```

Comment:
At some point I should try out a workflow which uses Node's type stripping capabilities and skips compilation (but not today).